frontend/src/__tests__/cypress/cypress/tests/e2e/storageClasses/storageClasses.cy.ts

```diff
@@ -76,7 +76,7 @@ describe('An admin user can manage Storage Classes from Settings -> Storage clas
   });
 
   it('An admin user can set an enabled Storage Class as the default one', () => {
-    cy.visitWithLogin('/', HTPASSWD_CLUSTER_ADMIN_USER);
+    cy.visitWithLogin('', HTPASSWD_CLUSTER_ADMIN_USER);
```

Comment:
@manosnoam I think using `/` as a relative URL or `/foo/` should be valid. Whether we need to strip trailing slashes should be something left to the underlying function.

Reply:
sure, I changed tests back to '/', and now passing the query parameters - only if defined in ODH_DASHOBAR_URL and if the relativeUrl is just '/' or empty.